presto-orc/src/main/java/com/facebook/presto/orc/AbstractOrcRecordReader.java

```diff
@@ -95,7 +95,7 @@
     private int nextBatchSize;
     private int maxBatchSize = MAX_BATCH_SIZE;
 
-    private final List<StripeInformation> stripes;
+    private final List<StripeInfo> stripes;
```

Comment:
How large each element would be? If we have a super wide table, would this cause heavy memory consumption? Shall we count memory for it? cc: @arunthirupathi